### DISCUSSION TITLE
.NET MAUI, how to load the iOS crash report to Visual Studio

### DISCUSSION THREAD

**Discussion by CsabaStupak** [dotnet/maui#15738] [Q&A] [answered]
### Description

I'm working on a .NET MAUI application. The project is ready and it works fine on Android. It even works fine on my iPhone - both the Debug and Release version. However when I publish the application to App Store, it is rejected because it is crashing. I received several crashlog-9850E490-EC2E-4278-A4EE-8EB08D14C815.txt like files. I'm not sure how to decode them so I can see the call-stack where the crash happens.

For Windows project I always have a *.dmp file what I can open using Visual Studio and see the call-stack. However I'm not sure how to open such iOS crash report using Visual Studio for Mac.

### Public API Changes

No API change.

### Intended Use-Case

See the description above.

**Comment by drasticactions** [comment]
Moving this to discussions, as it's not a proposal and seems specific to your application.

@rolfbjarne Do you have any tips to help this person?

**Comment by CsabaStupak** [comment, marked as answer]
It looks like the issue was with Profile certificate. Not sure if the certificate became invalid because of "new Apple program license agreement" or due to deep-link support I added to my application. 

Anyway this was pretty frustrating. The problem I see here that the compiler, the publish process and even the review or CER report did not warn about the certificate issue (it just crashed) and I just can guess what could be the problem and do trial-error approach to solve the issue... Here is a definitive room for improvement here in Visual Studio. The iOS development is several times longer comparing to Android due to this "Apple certificate hell".

In this case it took 2 weeks until I find that this is the problem :-(
